build/package/centos7-rpm.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:7
```

Comment:
How I wish we could pass the `FROM` in as a variable :/

Reply:
yep, I know there's some tooling to have templates but felt maybe overkill for now

Reply:
Yup agreed